components/lane_follower/src/individual_follower.py

```diff
@@ -0,0 +1,150 @@
+import math
+
+import cv2
+import numpy as np
+import toml
+from einops import repeat
+
+
+class ifResult:
+    def __init__(self):
+        self.result_img = None
+        self.empty_windows = 0
+        self.heading = 0
+        self.slope = 0
+
+    pass
+
+
+class IndividualFollower:
+
+    def __init__(self, toml_path='Follower_Consts.toml'):
+
+        self.consts = toml.load(toml_path)["Individual_Follower"]
+        # self.fit is set in Plot_Line
+        self.fit = None
+        # These two are set below
+        self._binary_warped = None
+        self._histogram = None
+
+    # Need to determine if these should remain properties
+    # Or if they should be passed as arguments to Plot_Line
+    @property
+    def _binary_warped(self, value: np.ndarray):
+        self._binary_warped = value
+        self._histogram = np.sum(
+            self.binary_warped[self.binary_warped.shape[0] // 2 :, :], axis=0
+        )
+
+    def get_white_pixels(self):
+        # Returns an output image
+        NWINDOWS = self.consts["NWINDOWS"]
+        SEARCH_WIDTH = self.consts['SEARCH_WIDTH']
```

Comment:
See, if this was a config class, we wouldn't need these intermediate variables that need to be made. We could also give default values to the config class so that we don't need to always pass it to `__init__()`